### ISSUE TITLE
Crash on launch with latest weekly (36480) on macOS 14.4

### ISSUE THREAD

**Issue by JohnOCFII** [FreeCAD/FreeCAD#13036]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem description

I just downloaded the fresh (7 minutes old) weekly.  I get an immediate crash on launch attempt.

Version is 36480.  Mac OS ARM.

Here's the first part of the crash dump:
```
-------------------------------------
Translated Report (Full Report Below)
-------------------------------------

Process:               freecad [10989]
Path:                  /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/bin/freecad
Identifier:            freecad
Version:               ???
Code Type:             ARM-64 (Native)
Parent Process:        Exited process [10983]
User ID:               501

Date/Time:             2024-03-18 18:24:04.6317 -0500
OS Version:            macOS 14.4 (23E214)
Report Version:        12
Anonymous UUID:        E1A9B3E8-17C0-3635-633F-31D076EC9805

Sleep/Wake UUID:       6010C1E9-7187-4F05-9E45-8DD4661AA6DF

Time Awake Since Boot: 100000 seconds
Time Since Wake:       2080 seconds

System Integrity Protection: enabled

Crashed Thread:        0  Dispatch queue: com.apple.main-thread

Exception Type:        EXC_BAD_ACCESS (SIGSEGV)
Exception Codes:       KERN_INVALID_ADDRESS at 0x0000000000000004
Exception Codes:       0x0000000000000001, 0x0000000000000004

Termination Reason:    Namespace SIGNAL, Code 11 Segmentation fault: 11
Terminating Process:   exc handler [10989]

VM Region Info: 0x4 is not in any region.  Bytes before following region: 4371791868
      REGION TYPE                    START - END         [ VSIZE] PRT/MAX SHRMOD  REGION DETAIL
      UNUSED SPACE AT START
--->  
      __TEXT                      104944000-104964000    [  128K] r-x/r-x SM=COW  /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/bin/freecad

Thread 0 Crashed::  Dispatch queue: com.apple.main-thread
0   libFreeCADGui.dylib           	       0x10677e024 Gui::MainWindow::_updateActions() + 688
1   libQt5Core.5.15.13.dylib      	       0x10864de84 0x108508000 + 1334916
2   libQt5Core.5.15.13.dylib      	       0x108631094 QCoreApplication::notifyInternal2(QObject*, QEvent*) + 176
3   libFreeCADGui.dylib           	       0x10638e658 Gui::StartupPostProcess::showMainWindow() + 312
4   libFreeCADGui.dylib           	       0x10638cfd8 Gui::StartupPostProcess::execute() + 232
5   libFreeCADGui.dylib           	       0x10629d934 Gui::Application::runApplication() + 1288
6   freecad                       	       0x10494b030 main + 4932
7   dyld                          	       0x193bce0e0 start + 2360

Thread 1:
0   libsystem_pthread.dylib       	       0x193f51d20 start_wqthread + 0

Thread 2:
0   libsystem_pthread.dylib       	       0x193f51d20 start_wqthread + 0

Thread 3:
0   libsystem_pthread.dylib       	       0x193f51d20 start_wqthread + 0

Thread 4:
0   libsystem_pthread.dylib       	       0x193f51d20 start_wqthread + 0

Thread 5:: com.apple.CFSocket.private
0   libsystem_kernel.dylib        	       0x193f21358 __select + 8
1   CoreFoundation                	       0x19405f30c __CFSocketManager + 640
2   libsystem_pthread.dylib       	       0x193f56f94 _pthread_start + 136
3   libsystem_pthread.dylib       	       0x193f51d34 thread_start + 8

Thread 6:: Thread (pooled)
0   libsystem_kernel.dylib        	       0x193f199ec __psynch_cvwait + 8
1   libsystem_pthread.dylib       	       0x193f5755c _pthread_cond_wait + 1228
2   libQt5Core.5.15.13.dylib      	       0x108528504 0x108508000 + 132356
3   libsystem_pthread.dylib       	       0x193f51d34 thread_start + 8

Thread 7:: Thread (pooled)
0   libsystem_kernel.dylib        	       0x193f199ec __psynch_cvwait + 8
1   libsystem_pthread.dylib       	       0x193f5755c _pthread_cond_wait + 1228
2   libQt5Core.5.15.13.dylib      	       0x108528504 0x108508000 + 132356
3   libsystem_pthread.dylib       	       0x193f51d34 thread_start + 8

Thread 8:: Thread (pooled)
0   libsystem_kernel.dylib        	       0x193f199ec __psynch_cvwait + 8
1   libsystem_pthread.dylib       	       0x193f5755c _pthread_cond_wait + 1228
2   libQt5Core.5.15.13.dylib      	       0x108528504 0x108508000 + 132356
3   libsystem_pthread.dylib       	       0x193f51d34 thread_start + 8

Thread 9:: Thread (pooled)
0   libsystem_kernel.dylib        	       0x193f199ec __psynch_cvwait + 8
1   libsystem_pthread.dylib       	       0x193f5755c _pthread_cond_wait + 1228
2   libQt5Core.5.15.13.dylib      	       0x108528504 0x108508000 + 132356
3   libsystem_pthread.dylib       	       0x193f51d34 thread_start + 8

Thread 10:: Thread (pooled)
0   libsystem_kernel.dylib        	       0x193f199ec __psynch_cvwait + 8
1   libsystem_pthread.dylib       	       0x193f5755c _pthread_cond_wait + 1228
2   libQt5Core.5.15.13.dylib      	       0x108528504 0x108508000 + 132356
3   libsystem_pthread.dylib       	       0x193f51d34 thread_start + 8

Thread 11:: Thread (pooled)
0   libsystem_kernel.dylib        	       0x193f199ec __psynch_cvwait + 8
1   libsystem_pthread.dylib       	       0x193f5755c _pthread_cond_wait + 1228
2   libQt5Core.5.15.13.dylib      	       0x108528504 0x108508000 + 132356
3   libsystem_pthread.dylib       	       0x193f51d34 thread_start + 8

Thread 12:: Thread (pooled)
0   libsystem_kernel.dylib        	       0x193f199ec __psynch_cvwait + 8
1   libsystem_pthread.dylib       	       0x193f5755c _pthread_cond_wait + 1228
2   libQt5Core.5.15.13.dylib      	       0x108528504 0x108508000 + 132356
3   libsystem_pthread.dylib       	       0x193f51d34 thread_start + 8

Thread 13:: Thread (pooled)
0   libsystem_kernel.dylib        	       0x193f199ec __psynch_cvwait + 8
1   libsystem_pthread.dylib       	       0x193f5755c _pthread_cond_wait + 1228
2   libQt5Core.5.15.13.dylib      	       0x108528504 0x108508000 + 132356
3   libsystem_pthread.dylib       	       0x193f51d34 thread_start + 8

Thread 14:: Thread (pooled)
0   libsystem_kernel.dylib        	       0x193f199ec __psynch_cvwait + 8
1   libsystem_pthread.dylib       	       0x193f5755c _pthread_cond_wait + 1228
2   libQt5Core.5.15.13.dylib      	       0x108528504 0x108508000 + 132356
3   libsystem_pthread.dylib       	       0x193f51d34 thread_start + 8

Thread 15:: Thread (pooled)
0   libsystem_kernel.dylib        	       0x193f199ec __psynch_cvwait + 8
1   libsystem_pthread.dylib       	       0x193f5755c _pthread_cond_wait + 1228
2   libQt5Core.5.15.13.dylib      	       0x108528504 0x108508000 + 132356
3   libsystem_pthread.dylib       	       0x193f51d34 thread_start + 8

Thread 16:: com.apple.NSEventThread
0   libsystem_kernel.dylib        	       0x193f161f4 mach_msg2_trap + 8
1   libsystem_kernel.dylib        	       0x193f28b24 mach_msg2_internal + 80
2   libsystem_kernel.dylib        	       0x193f1ee34 mach_msg_overwrite + 476
3   libsystem_kernel.dylib        	       0x193f16578 mach_msg + 24
4   CoreFoundation                	       0x194036058 __CFRunLoopServiceMachPort + 160
5   CoreFoundation                	       0x19403491c __CFRunLoopRun + 1208
6   CoreFoundation                	       0x194033e0c CFRunLoopRunSpecific + 608
7   AppKit                        	       0x1979b5cb4 _NSEventThread + 144
8   libsystem_pthread.dylib       	       0x193f56f94 _pthread_start + 136
9   libsystem_pthread.dylib       	       0x193f51d34 thread_start + 8


Thread 0 crashed with ARM Thread State (64-bit):
    x0: 0x0000000000000000   x1: 0x0000600000a6b7c0   x2: 0x000000016b4badf8   x3: 0x000000010677dd74
    x4: 0x0000000000000000   x5: 0x0000000000000690   x6: 0x000060000045ae20   x7: 0x0000000000000001
    x8: 0x0000600003622c00   x9: 0x0000000000000000  x10: 0x000000016b4ba53c  x11: 0x000000000000001e
   x12: 0x0000000000000122  x13: 0x0000000000000114  x14: 0x0000000000000122  x15: 0x000000000000012c
   x16: 0x00000001fbc63ac0  x17: 0x00000001fbc63ac0  x18: 0x0000000000000000  x19: 0x000000016b4badf8
   x20: 0x0000000000000001  x21: 0x0000000000000003  x22: 0x000000016b4ba5d8  x23: 0x00000001fbc63ac0
   x24: 0x00000001fbc63ac0  x25: 0x0000600003b2a350  x26: 0x0000600002e095c0  x27: 0x000000016b4badf8
   x28: 0x00000001fbc63ac0   fp: 0x000000016b4ba4e0   lr: 0x000000010864de84
    sp: 0x000000016b4ba340   pc: 0x000000010677e024 cpsr: 0x60001000
   far: 0x0000000000000004  esr: 0x92000006 (Data Abort) byte read Translation fault

Binary Images:
       0x10bed0000 -        0x10bedbfff libobjc-trampolines.dylib (*) <e8a1b184-0349-3c61-a119-6543eb038317> /usr/lib/libobjc-trampolines.dylib
       0x10ffc0000 -        0x10ffc7fff libqsvgicon.dylib (*) <c063b1a7-a9eb-30b9-b22c-645222ad8d49> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/plugins/iconengines/libqsvgicon.dylib
       0x11f6bc000 -        0x11f707fff libqwebp.dylib (*) <b4f5290c-e808-362c-8013-e7326809e220> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/plugins/imageformats/libqwebp.dylib
       0x10ffb0000 -        0x10ffb3fff libqwbmp.dylib (*) <6cb20edc-76aa-3619-8c3e-7d4c6ea26837> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/plugins/imageformats/libqwbmp.dylib
       0x11f658000 -        0x11f6abfff libqtiff.dylib (*) <653316c4-c02a-3ea0-84c5-7b5b9a1d886a> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/plugins/imageformats/libqtiff.dylib
       0x10fcf0000 -        0x10fcf3fff libqtga.dylib (*) <c1ee90e1-d32a-338f-b44a-a720df73ab6d> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/plugins/imageformats/libqtga.dylib
       0x10ff9c000 -        0x10ffa3fff libqsvg.dylib (*) <87164d5a-2739-3583-b5d8-998b0b9d1d13> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/plugins/imageformats/libqsvg.dylib
       0x10ff88000 -        0x10ff8ffff libqpdf.dylib (*) <79728259-deb2-3154-b3ae-9081cce6f5d0> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/plugins/imageformats/libqpdf.dylib
       0x13b000000 -        0x13b50bfff libQt5Pdf.5.15.13.dylib (*) <f6a1a642-079b-3fba-bb57-a9adfa123a67> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/lib/libQt5Pdf.5.15.13.dylib
       0x10ff74000 -        0x10ff7bfff libqmacjp2.dylib (*) <af7bd629-0ce1-36da-ba6c-b51efe39b6ee> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/plugins/imageformats/libqmacjp2.dylib
       0x10ff60000 -        0x10ff67fff libqmacheif.dylib (*) <bf91e2fe-f2a7-3bee-a807-7e40d861ba4b> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/plugins/imageformats/libqmacheif.dylib
       0x10ff4c000 -        0x10ff53fff libqjpeg.dylib (*) <94f5d6da-3188-3668-a5df-f8fa75e17ae3> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/plugins/imageformats/libqjpeg.dylib
       0x11f558000 -        0x11f5dffff libjpeg.8.3.2.dylib (*) <aed1b2f9-eac1-38ce-932a-987cde067790> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/lib/libjpeg.8.3.2.dylib
       0x10ff38000 -        0x10ff3ffff libqico.dylib (*) <dd55a847-4fd9-3a04-a83c-c3f73119279f> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/plugins/imageformats/libqico.dylib
       0x10fcdc000 -        0x10fce3fff libqicns.dylib (*) <01e14e3f-b9cf-3054-903d-0437c8379780> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/plugins/imageformats/libqicns.dylib
       0x10fcc8000 -        0x10fccffff libqgif.dylib (*) <27446dc0-3ea1-3b31-ba8b-9a16848002d7> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/plugins/imageformats/libqgif.dylib
       0x10ff00000 -        0x10ff1ffff libqmacstyle.dylib (*) <e90c7b61-6a58-3f7b-8932-e9b0c125755d> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/plugins/styles/libqmacstyle.dylib
       0x10bdcc000 -        0x10be37fff com.apple.AppleMetalOpenGLRenderer (1.0) <83d94df2-aca8-3695-a3a1-bf8ca16657ac> /System/Library/Extensions/AppleMetalOpenGLRenderer.bundle/Contents/MacOS/AppleMetalOpenGLRenderer
       0x10a370000 -        0x10a46ffff libqcocoa.dylib (*) <aab8d279-f17b-3cf9-a1e1-149160c432c0> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/plugins/platforms/libqcocoa.dylib
       0x108fa8000 -        0x108fe7fff libQt5DBus.5.15.13.dylib (*) <ec50cb40-8dec-329e-a263-ca81d5a9435f> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/lib/libQt5DBus.5.15.13.dylib
       0x104d3c000 -        0x104d47fff pyexpat.cpython-311-darwin.so (*) <7bcd29ee-4513-36e3-8173-c86d3af89e8f> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/lib/python3.11/lib-dynload/pyexpat.cpython-311-darwin.so
       0x104d28000 -        0x104d2ffff _struct.cpython-311-darwin.so (*) <d8684217-fd76-3539-9783-648f5260daac> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/lib/python3.11/lib-dynload/_struct.cpython-311-darwin.so
       0x104d14000 -        0x104d1bfff binascii.cpython-311-darwin.so (*) <644a8959-46ef-3854-9865-00bac0c8c6f2> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/lib/python3.11/lib-dynload/binascii.cpython-311-darwin.so
       0x104c9c000 -        0x104c9ffff _typing.cpython-311-darwin.so (*) <ba0370c7-6012-3246-9eed-1f3311d10789> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/lib/python3.11/lib-dynload/_typing.cpython-311-darwin.so
       0x104cf0000 -        0x104d03fff _datetime.cpython-311-darwin.so (*) <4bcb943a-00fc-3fb3-b17d-ca765bed305c> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/lib/python3.11/lib-dynload/_datetime.cpython-311-darwin.so
       0x104cb4000 -        0x104cbffff math.cpython-311-darwin.so (*) <128b957b-c0db-37dd-ab6c-38c404beb6f7> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/lib/python3.11/lib-dynload/math.cpython-311-darwin.so
       0x104c8c000 -        0x104c8ffff _opcode.cpython-311-darwin.so (*) <810122fa-0dbf-32fb-a4a2-898b0bfd7f1b> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/lib/python3.11/lib-dynload/_opcode.cpython-311-darwin.so
       0x10627c000 -        0x107223fff libFreeCADGui.dylib (*) <1aa6cf99-0407-3acf-b4dc-13f75116273f> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/lib/libFreeCADGui.dylib
       0x105320000 -        0x10566ffff libpython3.11.dylib (*) <4aeb17fc-97e5-30a7-86f0-223257713acb> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/lib/libpython3.11.dylib
       0x107740000 -        0x107acffff libFreeCADApp.dylib (*) <34058a16-3e09-30f1-99c3-b0cbd9bcb481> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/lib/libFreeCADApp.dylib
       0x104ff4000 -        0x10514bfff libFreeCADBase.dylib (*) <8e0da580-928b-375b-85e6-89a8b55da0d9> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/lib/libFreeCADBase.dylib
       0x105d00000 -        0x105fcbfff libQt5Widgets.5.15.13.dylib (*) <b8c6b625-4f46-3bc1-9323-14363f6d3080> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/lib/libQt5Widgets.5.15.13.dylib
       0x107cf4000 -        0x107f63fff libQt5Gui.5.15.13.dylib (*) <46e8db07-b44c-3ffb-9923-1b9b497c562d> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/lib/libQt5Gui.5.15.13.dylib
       0x108508000 -        0x1088b7fff libQt5Core.5.15.13.dylib (*) <177308f9-bdf9-322f-815c-7001203968b0> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/lib/libQt5Core.5.15.13.dylib
       0x1058c4000 -        0x10597ffff libc++.1.0.dylib (*) <d9229d7d-8f7e-3e8f-a88e-15125158bcae> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/lib/libc++.1.0.dylib
       0x109098000 -        0x109523fff libCoin.80.0.2.dylib (*) <0ea8b671-0783-3e38-9760-d04d48627df4> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/lib/libCoin.80.0.2.dylib
       0x104dec000 -        0x104e03fff libboost_filesystem.dylib (*) <f463f328-d246-3408-811a-22d1d07b6072> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/lib/libboost_filesystem.dylib
       0x104e64000 -        0x104e87fff libQt5PrintSupport.5.15.13.dylib (*) <9519b91d-7e78-39b1-8f0c-84f2b8863bf3> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/lib/libQt5PrintSupport.5.15.13.dylib
       0x104eb0000 -        0x104ed7fff libQt5Svg.5.15.13.dylib (*) <de375b14-0289-3241-be9d-1546686b7ca0> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/lib/libQt5Svg.5.15.13.dylib
       0x104efc000 -        0x104f23fff libpyside2.cpython-311-darwin.5.15.8.dylib (*) <911e76e0-514b-320a-832c-3a60d64e5fb2> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/lib/libpyside2.cpython-311-darwin.5.15.8.dylib
       0x108088000 -        0x108257fff libxerces-c-3.2.dylib (*) <e6ccb2b8-c22e-3ab0-a234-3843df8fc56c> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/lib/libxerces-c-3.2.dylib
       0x104f40000 -        0x104f5bfff libQt5Xml.5.15.13.dylib (*) <fa1b971f-40c4-3bd9-a2dc-0d4a5e9d9b8d> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/lib/libQt5Xml.5.15.13.dylib
       0x10522c000 -        0x10527bfff libshiboken2.cpython-311-darwin.5.15.8.dylib (*) <284baa34-10b6-3274-a324-97f687094720> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/lib/libshiboken2.cpython-311-darwin.5.15.8.dylib
       0x105ae4000 -        0x105b8ffff libQt5Network.5.15.13.dylib (*) <d70702ad-e7fe-3471-9ceb-34d6015bdd66> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/lib/libQt5Network.5.15.13.dylib
       0x10529c000 -        0x1052d3fff libboost_program_options.dylib (*) <ede54674-e484-3ca1-87ce-8b3b9d572a28> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/lib/libboost_program_options.dylib
       0x104e18000 -        0x104e2bfff libz.1.2.13.dylib (*) <4c6e9be5-0ad8-3dd3-b012-7ddc6c4f797f> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/lib/libz.1.2.13.dylib
       0x105be4000 -        0x105c7bfff libcurl.4.dylib (*) <7ddeb40d-321d-35e4-9a9f-7e71256a9216> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/lib/libcurl.4.dylib
       0x108b9c000 -        0x108d27fff libicuuc.73.2.dylib (*) <72a52987-9194-32bc-b778-64dfc256bc7f> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/lib/libicuuc.73.2.dylib
       0x104fa8000 -        0x104fcbfff libnghttp2.14.dylib (*) <2a82133e-9e89-3752-810b-5e9dd0d7e6ac> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/lib/libnghttp2.14.dylib
       0x105a2c000 -        0x105a5ffff libssh2.1.0.1.dylib (*) <48b6486e-329e-3601-8238-6ab0f3e815de> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/lib/libssh2.1.0.1.dylib
       0x108398000 -        0x10842bfff libssl.3.dylib (*) <8e65fbd1-f2b8-3573-bdee-2191ae67111f> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/lib/libssl.3.dylib
       0x109bdc000 -        0x109eebfff libcrypto.3.dylib (*) <b3f99b2e-54af-383d-bccb-845f50ec41ea> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/lib/libcrypto.3.dylib
       0x105ca4000 -        0x105ce3fff libgssapi_krb5.2.2.dylib (*) <5bfa4954-b295-320b-90a3-04727984e4db> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/lib/libgssapi_krb5.2.2.dylib
       0x108990000 -        0x108a4bfff libzstd.1.5.5.dylib (*) <4dba0bb4-b15e-3bae-8f6c-59dcae6b6ae8> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/lib/libzstd.1.5.5.dylib
       0x108a64000 -        0x108b03fff libkrb5.3.3.dylib (*) <5ded8aaf-3ad4-3cec-9976-efc4a85621c6> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/lib/libkrb5.3.3.dylib
       0x104e38000 -        0x104e4bfff libk5crypto.3.1.dylib (*) <21ef9d42-1b51-3677-bf77-39197bae18d4> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/lib/libk5crypto.3.1.dylib
       0x104dd0000 -        0x104dd3fff libcom_err.3.0.dylib (*) <029e1c76-0a04-3ab3-aa8c-727700a6fb3e> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/lib/libcom_err.3.0.dylib
       0x104f8c000 -        0x104f97fff libkrb5support.1.1.dylib (*) <50d73f23-cc95-34ae-9e9c-9b8f3d33aa0d> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/lib/libkrb5support.1.1.dylib
       0x10bee4000 -        0x10dd73fff libicudata.73.2.dylib (*) <792ebfca-026c-37d9-9117-e1e085ba10d3> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/lib/libicudata.73.2.dylib
       0x104fdc000 -        0x104fdffff libboost_atomic.dylib (*) <2a47055e-8e9c-3c90-907a-9bb61c101d59> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/lib/libboost_atomic.dylib
       0x10a014000 -        0x10a24ffff libicui18n.73.2.dylib (*) <5dbb7f6c-0af3-3748-891a-99e9fed9b144> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/lib/libicui18n.73.2.dylib
       0x105a74000 -        0x105a93fff libexpat.1.8.10.dylib (*) <9a4d0af4-915a-34be-94cd-6eeb2a6fc23f> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/lib/libexpat.1.8.10.dylib
       0x105aa4000 -        0x105acffff libpng16.16.dylib (*) <0731bc41-8d69-3a25-b7bd-82cd5d207d3a> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/lib/libpng16.16.dylib
       0x1097a8000 -        0x109997fff libQt5Qml.5.15.13.dylib (*) <84ff2120-9ece-3edb-bfdb-b32f7d82a275> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/lib/libQt5Qml.5.15.13.dylib
       0x104944000 -        0x104963fff freecad (*) <8fa3d9f3-ec02-3b7a-941b-640afbfaf511> /Applications/Other Apps/FreeCAD-36480.app/Contents/Resources/bin/freecad
       0x193bc8000 -        0x193c509db dyld (*) <8e1e5ee2-f89a-33a7-bb0a-74bdc06b7828> /usr/lib/dyld
               0x0 - 0xffffffffffffffff ??? (*) <00000000-0000-0000-0000-000000000000> ???
       0x193f50000 -        0x193f5cfff libsystem_pthread.dylib (*) <45239f06-cc53-36d0-9933-7776ac7ea2fa> /usr/lib/system/libsystem_pthread.dylib
       0x193f15000 -        0x193f4fffb libsystem_kernel.dylib (*) <1889ce0a-52e7-3122-8907-81af920ac472> /usr/lib/system/libsystem_kernel.dylib
       0x193fb8000 -        0x194490fff com.apple.CoreFoundation (6.9) <33908a83-098f-3437-973e-fb848c4f39df> /System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation
       0x197852000 -        0x198b8efff com.apple.AppKit (6.9) <8b85317d-d56a-3370-8b78-da6c1d0fb53c> /System/Library/Frameworks/AppKit.framework/Versions/C/AppKit

External Modification Summary:
  Calls made by other processes targeting this process:
    task_for_pid: 0
    thread_create: 0
    thread_set_state: 0
  Calls made by this process:
    task_for_pid: 0
    thread_create: 0
    thread_set_state: 0
  Calls made by all processes on this machine:
    task_for_pid: 0
    thread_create: 0
    thread_set_state: 0

VM Region Summary:
ReadOnly portion of Libraries: Total=956.6M resident=0K(0%) swapped_out_or_unallocated=956.6M(100%)
Writable regions: Total=1.5G written=0K(0%) resident=0K(0%) swapped_out=0K(0%) unallocated=1.5G(100%)

                                VIRTUAL   REGION 
REGION TYPE                        SIZE    COUNT (non-coalesced) 
===========                     =======  ======= 
Activity Tracing                   256K        1 
ColorSync                          640K       29 
CoreAnimation                      160K       10 
CoreGraphics                        32K        2 
CoreUI image data                  304K        1 
Foundation                          16K        1 
Kernel Alloc Once                   32K        1 
MALLOC                             1.5G       44 
MALLOC guard page                  192K       12 
STACK GUARD                       56.3M       17 
Stack                             16.5M       17 
VM_ALLOCATE                       5280K       15 
__AUTH                             972K      214 
__AUTH_CONST                      16.1M      381 
__CTF                               824        1 
__DATA                            7340K      441 
__DATA_CONST                      21.6M      446 
__DATA_DIRTY                       931K      117 
__FONT_DATA                        2352        1 
__GLSLBUILTINS                    5174K        1 
__LINKEDIT                       545.6M       66 
__OBJC_RO                         71.7M        2 
__OBJC_RW                         2195K        1 
__TEXT                           411.0M      461 
dyld private memory                272K        2 
mapped file                      185.6M       27 
shared memory                     1392K       16 
===========                     =======  ======= 
TOTAL                              2.8G     2327 

```

### Full version info

```shell
(Can't get that far) Fresh weekly 0.22 - 36480
```

### Subproject(s) affected?

None

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by maxwxyz** [comment]
Should be fixed with #13033

**Comment by JohnOCFII** [comment]
Confirmed resolved with build: 36487.

```
OS: macOS 14.4
Word size of FreeCAD: 64-bit
Version: 0.22.0dev.36487 (Git)
Build type: Release
Branch: main
Hash: f5baef888c25937026ee50001a4fe4293a9f11ee
Python 3.11.8, Qt 5.15.13, Coin 4.0.2, Vtk 9.2.6, OCC 7.7.2
Locale: C/Default (C)
Installed mods: 
  * OpenLight_Beta 2023.12.17
  * FeedsAndSpeeds 0.5.0
  * OpenDark 2023.12.17
```
